knack/arguments.py

```diff
@@ -300,8 +302,10 @@ def __call__(self, parser, namespace, values, option_string=None):
             return ExperimentalArgumentAction
 
         def _get_experimental_arg_message(self):
-            return "{} '{}' is experimental and not covered by customer support. " \
-                   "Please use with discretion.".format(self.object_type.capitalize(), self.target)
+            from .experimental import EXPERIMENTAL_MESSAGE
```

Comment:
How could developers customize the message in your current design? It seems that you define the message variable with hard-coded message. If I am a developer who want to change the message, I have to re-write the message in knack. Is it by design?

Reply:
A developer can customize the message like https://github.com/Azure/azure-cli/pull/15730. I will add a comment to the variable as well.